src/avfoundation.cs

```diff
@@ -9642,274 +9877,319 @@ enum AVAuthorizationMediaType {
 		Audio,
 	}
 
-	[NoWatch]
```

Comment:
I'm not sure it's correct - the registrar will still see (and generate code for) this type and Apple could reject that. c.c. @rolfbjarne 

I think a better way is to have the type `[Static]` on watchOS (since the field is `static` binary code will remain compatible). That way there will not be trace of the type in native code.

_note: remove `#if` is nice - but it does not achieve the same result here_

Reply:
This is what **API diff** shows on watchOS

```
New Type AVFoundation.AVCaptureDevice

public class AVCaptureDevice : Foundation.NSObject, Foundation.INSObjectProtocol, ObjCRuntime.INativeObject, System.IDisposable, System.IEquatable<Foundation.NSObject> {
	// constructors
	protected AVCaptureDevice (Foundation.NSObjectFlag t);
	protected AVCaptureDevice (IntPtr handle);
	// properties
	public override IntPtr ClassHandle { get; }
	public static float LensPositionCurrent { get; }
}
```